config/monitoring/prometheus/apps/prometheus-configs.yaml

```diff
@@ -1577,3 +1602,92 @@ data:
           labels:
             severity: warning
             instance: trustyai-service-operator-controller-manager
+
+  model-registry-operator-recording.rules: |
+    groups:
+      - name: SLOs - Model Registry Operator
+        rules:
+        - expr: |
+            absent(up{job=~'Model Registry Operator'}) * 0 or vector(1)
+          labels:
+            instance: model-registry-operator
+          record: probe_success
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="model-registry-operator"}[1d]))
+          labels:
+            instance: model-registry-operator
+          record: probe_success:burnrate1d
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="model-registry-operator"}[1h]))
+          labels:
+            instance: model-registry-operator
+          record: probe_success:burnrate1h
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="model-registry-operator"}[2h]))
+          labels:
+            instance: model-registry-operator
+          record: probe_success:burnrate2h
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="model-registry-operator"}[30m]))
+          labels:
+            instance: model-registry-operator
+          record: probe_success:burnrate30m
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="model-registry-operator"}[3d]))
+          labels:
+            instance: model-registry-operator
+          record: probe_success:burnrate3d
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="model-registry-operator"}[5m]))
+          labels:
+            instance: model-registry-operator
+          record: probe_success:burnrate5m
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="model-registry-operator"}[6h]))
+          labels:
+            instance: model-registry-operator
+          record: probe_success:burnrate6h
+
+  model-registry-operator-alerting.rules: |
+    groups:
+      - name: SLOs-probe_success_model_controller
+        rules:
+        - alert: Model Registry Operator Probe Success Burn Rate
+          annotations:
+            message: 'High error budget burn for {{ $labels.instance }} (current value: {{ $value }}).'
+            triage: "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Model-Serving/rhods-model-registry-operator-probe-success-burn-rate.md"
```

Comment:
one thing before merge this PR is to get https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Model-Serving/rhods-model-registry-operator-probe-success-burn-rate.md created.
preferably to name the file as `rhoai-model-registry-operator-probe-success-burn-rate.md`

Reply:
Related to that, I used the simple name `Model Registry Operator` in the config names and descriptions. I noticed that some components used ODH prefix. Does model registry need to use `RHOAI Model Registry Operator` instead?

Reply:
think you can make a PR and ask (someone) in the project to review/approve it.
should be a owner file somewhere with name list there

Reply:
tbh, i do not know the answer. I would say if ODH already exists by other component, then you can use it, or just skip ODH or any downstream name. but better check with project, if any hard/soft rules.